content/en/blog/_posts/2021-06-18-Announcing-Kubernetes-Upstream-Community-Group-Annual-Reports.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: blog
+title: "Announcing Kubernetes Upstream Community Group Annual Reports"
```

Comment:
Optional (and untested) tweak that should show up in search engine indexes, maybe also feeds:
```suggestion
title: "Announcing Kubernetes Upstream Community Group Annual Reports"
description: >
  Introducing brand new Kubernetes Community Group Annual Reports for
  Special Interest Groups and Working Groups.
```

Reply:
+1 on adding a description, but personally I'd remove the "brand new" part of the description.^^;;

Reply:
```suggestion
title: "Announcing Kubernetes Community Group Annual Reports"
```
As this is the upstream blog, I don't think its necessarily needed in the title 👍 
**NOTE:** This would also mean a file name change